### PR TITLE
Fixes Reqtorio formatting

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Title and as per Tivi's request.
## Why It's Good For The Game
Consistent formatting good.
## Changelog
:cl:
fix: fixes reqtorio formatting.
/:cl:

### CODE REVIEW

File: code/modules/factory/parts.dm [hunk 1]
```diff
@@ -216,7 +216,7 @@ GLOBAL_LIST_INIT(IFF_ammo, list(
 
 /obj/item/factory_part/auto_sniper_magazine
 	name = "IFF high caliber bullet box"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:218]: these need  \improper in front

File: code/modules/factory/parts.dm [hunk 2]
```diff
@@ -366,7 +366,7 @@ GLOBAL_LIST_INIT(swat_mask, list(
 
 /obj/item/factory_part/swat_mask
 	name = "SWAT mask"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:368]: here too

File: code/modules/factory/parts.dm [hunk 3]
```diff
@@ -415,17 +415,17 @@ GLOBAL_LIST_INIT(module, list(
 	))
 
 /obj/item/factory_part/module_valk
-	name = "Valkyrie Automedical Armor System"
-	desc = "An unfinished Valkyrie Automedical Armor System module"
+	name = "Valkyrie automedical armor system"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:418]: improper or lowercase

game only cares about the first letter of names rest doesnt matter

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/factory/parts.dm (hunk 1 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm]: Done

File: code/modules/factory/parts.dm (hunk 2 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm]: Done

File: code/modules/factory/parts.dm (hunk 3 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm]: Done

File: code/modules/factory/parts.dm [hunk 4]
```diff
@@ -62,7 +62,7 @@ GLOBAL_LIST_INIT(bignade_recipe,  list(
 
 /obj/item/factory_part/bignade
 	name = "M15 grenade assembly"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:64]:
```suggestion
	name = "\improper M15 grenade assembly"
```

File: code/modules/factory/parts.dm [hunk 5]
```diff
@@ -160,16 +160,16 @@ GLOBAL_LIST_INIT(recoilless_missile_recipe, list(
 	recipe = GLOB.recoilless_missile_recipe
 
 /obj/item/factory_part/cloak_rr_missile
-	name = "Standard Recoilless ammo assembly"
-	desc = "An unfinished squat missile. It has a particularily large warhead."
+	name = "standard recoilless ammo assembly"
+	desc = "ASn unfinished squat missile. It has a particularily large warhead."
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:164]:
```suggestion
	desc = "An unfinished squat missile. It has a particularly large warhead."
```

File: code/modules/factory/parts.dm [hunk 6]
```diff
@@ -515,7 +516,7 @@ GLOBAL_LIST_INIT(mlrs_rocket, list(
 
 /obj/item/factory_part/mlrs_rocket
 	name = "MLRS rocket"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:518]:
```suggestion
	name = "\improper MLRS rocket"
```

File: code/modules/factory/parts.dm [hunk 7]
```diff
@@ -532,7 +533,7 @@ GLOBAL_LIST_INIT(thermobaric_wp_recipe, list(
 
 /obj/item/factory_part/thermobaric_wp
 	name = "RL-57 Thermobaric WP rocket array"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:535]:
```suggestion
	name = "\improper RL-57 Thermobaric WP rocket array"
```

File: code/modules/factory/parts.dm [hunk 8]
```diff
@@ -436,31 +436,32 @@ GLOBAL_LIST_INIT(module, list(
 	recipe = GLOB.module
 
 /obj/item/factory_part/module_tyr2
-	name = "Mark 2 Tyr Armor Reinforcement"
-	desc = "An unfinished Mark 2 Tyr Armor Reinforcement module"
+	name = "Mark 2 Tyr armor reinforcement"
```
Comment (TiviPlus) [inline code/modules/factory/parts.dm:439]:
```suggestion
	name = "\improper Mark 2 Tyr armor reinforcement"
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/factory/parts.dm (hunk 4 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm:64]: Done

File: code/modules/factory/parts.dm (hunk 5 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm:164]: Done

File: code/modules/factory/parts.dm (hunk 6 above)
Comment (Naaanii) [reply code/modules/factory/parts.dm:518]: Done

**Pull request merged**